src/thor/isochrone.cc

```diff
@@ -239,14 +239,15 @@ void Isochrone::ExpandForward(GraphReader& graphreader,
       continue;
     }
 
+    auto todo = nullptr;
     bool has_time_restrictions = false;
     // Check if the edge is allowed or if a restriction occurs. Get the edge speed.
     if (has_date_time_) {
       // With date time we check time dependent restrictions and access as well as get
       // traffic based speed if it exists
       if (!costing_->Allowed(directededge, pred, tile, edgeid, localtime, nodeinfo->timezone(),
                              has_time_restrictions) ||
-          costing_->Restricted(directededge, pred, edgelabels_, tile, edgeid, true, localtime,
+          costing_->Restricted(directededge, pred, edgelabels_, tile, edgeid, true, todo, localtime,
```

Comment:
I wasn't sure if there's any caveats about enabling this for isochrone or if it even matters so left out this behaviour via use of `nullptr`.

Reply:
Also I'm not aware of a easy way of unittesting isochrone, open to suggestions if we want this.